app/models/UserLogin.model.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * This class extends Database, this class check credentials.
+ */
+class UserLogin extends Database {
+  /**
+   *  This function check user's credentials.
+   *
+   * @param  string $email
+   *   User's email.
+   * @param  string $pass
+   *   User's password.
+   *
+   * @return string
+   *   Return result string.
+   */
+  public function check_credential(string $email, string $pass):string {
```

Comment:
In some of the methods we are using snake case and in others we are using camel case. Why is that ?